SECP-service/src/main/java/com/visucius/secp/DTO/GroupModifyRequest.java

```diff
@@ -0,0 +1,35 @@
+package com.visucius.secp.DTO;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class GroupModifyRequest {
+
+    @JsonProperty
+    public Set<Long> remove_roles = new HashSet<>();
```

Comment:
This class should just have roles and permission. In the Group resource, then you'll have POST /modify/id/roles and DELETE /modify/id/roles. same goes for the permissions